triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -413,6 +413,11 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers with Postg
       } yield succeed
   }
 
+  // TODO(SF, 2020-06-05): This test is temporarily disabled as too
+  // fragile. There is a race condition on the trigger restart and the
+  // network being unavailable (see
+  // https://dev.azure.com/digitalasset/adadc18a-d7df-446a-aacb-86042c1619c6/_apis/build/builds/45230/logs/124).
+  /*
```

Comment:
I think you can use `ignore` instead of commenting out, e.g. described here: https://www.scalatest.org/user_guide/tagging_your_tests